src/schema/v2/artwork/__tests__/comparableAuctionResults.test.ts

```diff
@@ -0,0 +1,124 @@
+import { runQuery } from "schema/v2/test/utils"
+import sinon from "sinon"
+
+describe("ComparableAuctionResults", () => {
+  let context = {}
+
+  beforeEach(() => {
+    context = {
+      artworkLoader: sinon
+        .stub()
+        .withArgs(mockArtwork.id)
+        .returns(Promise.resolve(mockArtwork)),
```

Comment:
(non-blocking heads up!) It's hard to avoid older copy pasta stuff in MP, but we shouldn't need to use sinon. [Jest idioms + simple functions work just fine and are simpler](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/__tests__/artistRecommendations.test.ts#L23-L35).